src/applications/personalization/preferences/helpers.jsx

```diff
@@ -1,56 +1,234 @@
+import React from 'react';
+
+const healthFAQ = () => (
+  <div>
+    <ul>
+      <li>
+        <a href="/health-care/eligibility/">
+          See if you’re eligible for VA health care benefits.
+        </a>
+      </li>
+      <li>
+        <a href="/health-care/how-to-apply/">
+          Learn how to apply online, by mail, or in person.
+        </a>
+      </li>
+      <li>
+        <a href="/health-care/about-va-health-benefits/">
+          Find out what kinds of health care and services are covered.
+        </a>
+      </li>
+    </ul>
+  </div>
+);
+
+const disabilityFAQ = () => (
+  <div>
```

Comment:
we probably don't need the wrapper `div`

Reply:
It might be hard to notice, but the div is necessary in this case, as there are two lists.